deploy/examples/create-external-cluster-resources.py

```diff
@@ -58,9 +58,11 @@
 try:
     # for 2.7.x
     from urlparse import urlparse
+    from urllib import urlencode as urlencode
```

Comment:
have you dry and run and tested the package?

This package looks common, i guess we can separate it out
>    from urlparse import urlparse

Reply:
@travisn @parth-gr In fact, the reason behind the change is in fact I'm getting this error on other PRs:

```
Run pylint $(git ls-files '*.py') -E
************* Module create-external-cluster-resources
deploy/examples/create-external-cluster-resources.py:1366:33: E0601: Using variable 'urllib' before assignment (used-before-assignment)
Error: Process completed with exit code 2.

```
I can't tell how it's directly related but my change seems to fix the `pylint` error. Besides, the `urlenconde` method as is being called will break in python2.7 as it's not available in `urllib.parse` instead, this method is on `urlib` directly.